metadata/src/main/java/org/apache/kafka/metadata/bootstrap/BootstrapDirectory.java

```diff
@@ -86,12 +84,7 @@ BootstrapMetadata readFromConfiguration() {
             return BootstrapMetadata.fromVersion(MetadataVersion.latestProduction(), "the default bootstrap");
         }
         MetadataVersion version = MetadataVersion.fromVersionString(ibp.get());
-        if (version.isLessThan(MINIMUM_BOOTSTRAP_VERSION)) {
```

Comment:
@cmccabe Is it important to automatically fallback if older metadata versions are received here or can we throw an error? It's harder to fallback (but not impossible) since I removed older metadata versions.

Reply:
Sure, will push a commit with this change.

Reply:
I guess this means the answer to my question is that the fallback is not important.

Reply:
yes, we're getting rid of the fallback